torch/export/unflatten.py

```diff
@@ -601,6 +601,12 @@ def finalize_outputs(self):
         if parent_out is None:
             return
 
+        parent_out.meta["val"] = (
+            graph_outputs.meta.get("val")
+            if isinstance(graph_outputs, torch.fx.Node)
+            else [o.meta.get("val") for o in graph_outputs]
+        )
+
```

Comment:
I went ahead to apply this change, and it seems to miss the nodes generated in line 615. Looks like the nodes generated by Proxy do not have any metadata, maybe we should add 

```python
proxy_out.meta = copy.copy(orig_output.meta)
```